samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Helpers/TestContext.cs

```diff
@@ -205,7 +205,7 @@ protected virtual void Dispose(bool disposing)
                                                             Action resendRequest = default)
         {
             int retry = 0;
-            while (retry < MaxRetries)
+            while (true)
```

Comment:
Was this change related to the error change?

Reply:
no, it wasn't. just a little refactoring. 
I thought did this change already but it seems it was lost in-between merges.